### ISSUE TITLE
Cogvideox_1_5_5b_I2V_01 workflow not working!

### ISSUE THREAD

**Issue by FranM2030** [kijai/ComfyUI-CogVideoXWrapper#255]
So I wanted to try out the example Cogvideox_1_5_5b_I2V_01.json workflow

I drop it in an up to date comfyui and get this missing node

![Screenshot 2024-11-20 110959](https://github.com/user-attachments/assets/2671e3a6-2619-423e-8f1e-3d2318645600)

it appears it doesn't exist so I delete and then get this

![Screenshot 2024-11-20 111319](https://github.com/user-attachments/assets/2ac70fe6-f794-420c-a031-6914176f9304)

turns out almost all of the settings and wiring is wrong, so I set everything up in the workflow that's supposed to explain to me how everything works

![Screenshot 2024-11-20 111442](https://github.com/user-attachments/assets/b68ac8a0-86bf-484f-a1a2-62bf44549b3a)

eventually I get to this point where it seems it can't download the model and can't start the generation

![Screenshot 2024-11-20 111920](https://github.com/user-attachments/assets/a8bd4481-6092-4391-84f2-9f26ac10dbe8)

So my questions are: 

1. Should the node download the model automatically?
2. The model appears to be Cogvideox_1_5_5b_I2V_01 I found a similar model on hugginface, so where do I actually put it? can't find a potential location
3. What else do I need to setup to get this workflow to work?

Anyone else got this to work???

cheers

**Comment by kijai** [comment]
That just looks like you either updated and didn't restart/refresh the browser so it looks for the old nodes still, or the nodes just didn't update and you loaded new workflow with old nodes.

**Comment by KrakeyMTL** [comment]
Here is a saved one from my server in case you want to play with it. 
NOTE: I'm on a L40 gpu so I have fast bits and super junk enabled!!!! the gpu can spit out a video using 1.5B in less than 8min using these settings ~ omg!! XD

[workflow_1.5B_working_L40_Server_reupload.json](https://github.com/user-attachments/files/17823145/workflow_1.5B_working_L40_Server_reupload.json)

**Comment by kijai** [comment]
When node definitions change while you have comfy up in a browser tab, you **always** have to refresh to get the new ones.

**Comment by FranM2030** [comment]
> Same happened here on the refresh - I needed to WIPE the workspace after my update, (fresh update fyi), download the RAW json text, make a notepad version locally, save it, then it loaded.
> 
> Why it was this annoying when usually you can just drag and drop - I dunno lol
> 
> But it does in fact work doing it like I did above. I just happened to make a thread on it too so can confirm working.

yes, after updating again, clearing the workflow, manually opening the new workflow, it would finally work and download the new model
